scripts/addMetamaskContractMapToHumanizerConsts.js

```diff
@@ -0,0 +1,35 @@
+const fsPromises = require('fs').promises
+const path = require('path')
+const humanizerInfo = require('../src/consts/humanizer/humanizerInfo.json')
+
+const humanizerInfoPath = path.join(
+  __dirname,
+  '..',
+  'src',
+  'consts',
+  'humanizer',
+  'humanizerInfo.json'
+)
+fetch('https://raw.githubusercontent.com/MetaMask/contract-metadata/master/contract-map.json')
+  .then((r) => r.json())
+  .then((mmContractMap) => {
+    Object.entries(mmContractMap).forEach(([address, info]) => {
```

Comment:
Might be overkill, sharing it as just an idea. Consider validating the structure of the `mmContractMap` and info fields (such as checking if name, symbol, decimals, etc. exist) to prevent any runtime errors if the data format changes unexpectedly.